include/ipmitool/ipmi_quantaoem.h

```diff
@@ -40,8 +40,10 @@
 #include <ipmitool/ipmi_sdr.h>
 
 #define OEM_QCT_NETFN 0x36
-#define OEM_QCT_GET_INFO 0x65
+#define OEM_QCT_NETFN_30 0x30
```

Comment:
This is a really bad name for a macro.
Looks like this must be put into `include/ipmitool/ipmi.h` and called like `IPMI_NETFM_CSOEM` for `/* Controller-specific OEM/Group function */`.

The number `0x30` is currently hijacked in that header by `IPMI_NETFN_TSOL`, but the latter must be redefined using that generic new macro. In future it will need to be moved into ipmi_tyanoem.h or something like that, but let's keep it where it is so far, just redefine.